components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -24,11 +24,14 @@ export default function GitAuth() {
 export const OrgSettingsPageWrapper: FunctionComponent = ({ children }) => {
     const currentOrg = useCurrentOrg();
 
+    const title = "Git Integrations";
+    const subtitle = "Configure Git integrations for self-managed instances of GitLab, GitHub, or Bitbucket.";
```

Comment:
Let's go with something like `Configure Git Auth for GitLab, or Github.`
Minus `self-managed` as this is to support SaaS (gitlab.com + github.com)